contracts/GuestList.vy

```diff
@@ -26,6 +26,7 @@ struct ProtectedLiquidity:
 interface Vault:
     def balanceOf(user: address) -> uint256: view
     def getPricePerFullShare() -> uint256: view
```

Comment:
This is gonna be fun when the new Vault changes this endpoint 😬

Reply:
This is only used with the old yYFI vault.